app/src/test/java/org/simple/clinic/teleconsultlog/teleconsultrecord/screen/TeleconsultRecordInitTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord.screen
+
+import com.spotify.mobius.test.FirstMatchers.hasEffects
+import com.spotify.mobius.test.FirstMatchers.hasModel
+import com.spotify.mobius.test.InitSpec
+import com.spotify.mobius.test.InitSpec.assertThatFirst
+import org.junit.Test
+import java.util.UUID
+
+class TeleconsultRecordInitTest {
+
+  @Test
+  fun `when screen is created and record id is not null, then load teleconsult record with prescribed drugs`() {
```

Comment:
Can we not use implementation specific terms like `record id is not null` in the name of the test please?

Reply:
Oh yes, I forgot to rename the test. We won't ever have a scenario with teleconsult record id being null.